src/Analysis/Problem/Vanilla/OverworldSettingsMissingProblem.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Aternos\Codex\Minecraft\Analysis\Problem\Vanilla;
+
+use Aternos\Codex\Minecraft\Analysis\Solution\Vanilla\GenerateNewWorldSolution;
+use Aternos\Codex\Minecraft\Translator\Translator;
+
+class OverworldSettingsMissingProblem extends VanillaProblem
+{
+    /**
+     * Get a human-readable message
+     *
+     * @return string
+     */
+    public function getMessage(): string
+    {
+        return Translator::getInstance()->getTranslation("overworld-settings-missing-problem");
+    }
+
+    /**
+     * Get an array of possible patterns
+     *
+     * The array key of the pattern will be passed to setMatches()
+     *
+     * @return string[]
+     */
+    public static function getPatterns(): array
+    {
+        return ['/Overworld settings missing/'];
```

Comment:
The full error message is always:

```
[...] [main/WARN] [net.minecraft.server.Main/]: Failed to load datapacks, can't proceed with server load. You can either fix your datapacks or reset to vanilla with --safeMode
java.util.concurrent.ExecutionException: java.lang.IllegalStateException: Overworld settings missing
```

right?

Maybe we should be a little more specific about the pattern for this problem.
Just to make sure we don't accidentally show a problem because we find this pattern in the logs, just because a mod, plugin, ... writes "Overworld settings missing" to the log.